### ISSUE TITLE
Possible lumen support 

### ISSUE THREAD

**Issue by dziurka** [lorisleiva/laravel-actions#32]
Is the package compatible with lumen? Have you ever thought about support for this framework?
Incidentally very good work ;)

**Comment by lorisleiva** [comment]
Hi there 👋

I always try my best not to use Laravel-only features in my packages so my guess is that it should be compatible with Lumen but to be completely honest I haven't yet used it within a Lumen application so I can't be 100% sure.

Let me know if you get into any gotchas and I'd be happy to make some adjustments to support Lumen.

**Comment by alignwebs** [comment]
Hi @lorisleiva 

While trying to create a new Action via command line on Lumen, I am getting this error 
![image](https://user-images.githubusercontent.com/7644807/92463700-c7347800-f1e9-11ea-9919-10c67e90f906.png)

Can we fix this error ?

**Comment by lorisleiva** [comment]
Hi there 👋 

Thanks for raising this. Yes this should be as simple as wrapping the `Route::actions()` macro inside a conditional that check if that method exists.

I'll leave this open until I have time to sort this out or someone else submits a PR.

Thanks again! 🙂

**Comment by alignwebs** [comment]
Thanks for responding and reopening the issue!

Let me try to submit PR :)

**Comment by lalitj** [comment]
I am seeing following error:
![Screenshot_2](https://user-images.githubusercontent.com/990271/95057670-210a5e00-0714-11eb-8943-86d3327562a3.png)
laravel-actions: 1.1

**Comment by lorisleiva** [comment]
Does Lumen not have an Event bus by default?

**Comment by lorisleiva** [comment]
Closing this due to inactivity. Feel free to open a new one for v2 if the problem persists.

**Comment by Hector1567XD** [comment]
> Hi there 👋
> 
> Thanks for raising this. Yes this should be as simple as wrapping the `Route::actions()` macro inside a conditional that check if that method exists.
> 
> I'll leave this open until I have time to sort this out or someone else submits a PR.
> 
> Thanks again! 🙂

Good afternoon 👀!

Even being a newbie I have challenged myself to use this package in Lumen xd, I have run into the error described in this post.

I had to modify the ActionServiceProvider.php to add `method_exists(Route::class, 'actions')` as suggested

![image](https://user-images.githubusercontent.com/22270579/112377599-556ed800-8cbc-11eb-914b-5f871aed7953.png)

But I think this will be a problem for me when I deploy to my server and run "composer update" as I should download the package without the modification.

Should I create a "Pull Request" to the package and when they approve it when deploying it, the modification should already be there? or is there any other way to leave this change there even on my server?

Also, would adding `method_exists(Route::class, 'actions')` affect the performance or functionality of the package in any way?

Thank you very much in advance!!!
